server/bin/pbench-audit-server.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import os
+import sys
+import glob
+import re
+import errno
+import hashlib
+import shutil
+import tempfile
+import tarfile
+from enum import Enum
+from argparse import ArgumentParser
+from s3backup import S3Config, Entry
+
+from pbench import PbenchConfig, BadConfig, get_pbench_logger, md5sum
+from pbench.report import Report
+
+
+_NAME_ = "pbench-audit-server"
+
+def verify_subdirs(config, controller, cnt):
+
+    linkdirs = sorted(config.LINKDIRS.split(' '))
+
+    states = glob.iglob(os.path.join(controller,'*'))
+    tarball, symlink, unexpected_objects, unexpected_dirs = ([] for i in range(4))
+
+    # Checking if the list is empty or not
+    if len(os.listdir(controller)) == 0:
+        print("\t* No state directories found in this controller directory.")
+    else:
+        for state in states:
+            st = os.path.basename(os.path.normpath(state))
+            if st != '_QUARANTINED' and ('WONT-INDEX' not in st):
+                if os.path.isdir(state):
+                    if st not in linkdirs:
+                        unexpected_dirs.append(st)
```

Comment:
Once you know it is a directory, there is no point allowing the code below to be executed as well, so using a `continue` statement here helps the reader understand the behavior.